device/mellanox/x86_64-mlnx_msn2700-r0/psu_sensors_conf_updater

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+function update_psu_sensors_configuration() {
+    local SENSORS_CONFIG_FILE="/tmp/sensors.conf"
+    local PSU_SENSORS_CONFIG="/usr/share/sonic/platform/psu_sensors.conf"
+
+    if [ -f $SENSORS_CONFIG_FILE ]; then
+        rm $SENSORS_CONFIG_FILE
+    fi
+
+    if [ -z "$1" ]; then
+        echo "ERROR: no sensors configuration file had been selected"
+        return
+    else
+        cp $1 $SENSORS_CONFIG_FILE
+    fi
+
+    platform=$(cat /usr/share/sonic/platform/platform.json | grep -m 1 "name" | cut -d '"' -f 4)
```

Comment:
can we use the full platform name and then we don't need to extract action? It will be simple in the future to add a new platform?

Reply:
I have replaced this operation by jq